api/app/Http/Middleware/Authenticate.php

```diff
@@ -53,6 +60,14 @@ public function handle(Request $request, Closure $next)
                 "message" => "User with specified hash was not found."
             ], 403);
         }
+    
+        if(($user->getRoleId() !== $this->roles->Admin) && $user->isBlocked())
+        {
+            http_response_code(403);
+            return response()->json([
+                "message" => "User is blocked."
```

Comment:
Create some error model to use everywhere

Reply:
created